src/components/contributions/contributions.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import EditIcon from './edit-icon.svg'
+
+const Contributions = ({ data }) => {
+  const commit = data.github.repository.object.history.nodes[0]
+  const file = data.github.repository.object.file
+  const lastUpdated = new Date(commit.author.date).toLocaleString('en-AU', {
+    dateStyle: 'long',
+  })
+  const editLink = `https://github.com/NUbots/NUbook/edit/main/${file.path}`
+  return (
+    <div className='text-sm text-secondary dark:text-secondary-inverted mb-6 border-t-2 border-b-2 border-gray-200 dark:border-gray-800 py-3 flex'>
+      <div className='flex-grow'></div>
+      <div className='pr-3 capitalize'>
+        <a
+          href={commit.url}
+          target='_blank'
+          className='lg:normal-case'
+          rel='noreferrer'
+        >
+          <span className='hidden lg:inline'>Last</span> updated{' '}
+          <span className='text-nubots-700 dark:text-nubots-500'>
+            {lastUpdated}
+          </span>
+        </a>
+      </div>
+      <div className='border-l-2 dark:border-gray-800 pl-3'>
+        <a href={editLink} className='flex items-center w-full'>
+          <EditIcon className='h-5' />
+          <span className='pl-1 hidden lg:block text-nubots-700 dark:text-nubots-500'>
+            Edit
+          </span>
+        </a>
+      </div>
+    </div>
+  )
+}
+
+Contributions.propTypes = {
```

Comment:
See comment below about passing through less props, I think just the latest commit and the url need to be passed through here rather than the entire graphql response. This will also mean less updates if the graphql API changes in the future. This also means that the prop names can be more descriptive than `data`